cypress/integration/visitorCanSeeSingleArticle.feature.js

```diff
@@ -16,14 +16,14 @@ describe("Visitor can see a single article", () => {
   });
 
   it("is expected to display correct url", () => {
-    cy.url().should("contain", "360");
+    cy.url().should("contain", "1");
   });
 
   it("is expected to see article title", () => {
-    cy.get("[data-cy=article-title]").should("contain", "Latin News");
+    cy.get("[data-cy=article-title]").should("contain", "The Grapes of Wrath");
   });
 
   it("is expected to see article published date", () => {
-    cy.get("[data-cy=article-created]").should("contain", "2022-01-13");
+    cy.get("[data-cy=article-created]").should("contain", "2022-01-23");
```

Comment:
You can check if the full date is displayed, from the fixture

Reply:
I don't think it is nececary. 

The important thing here is that it "contains" => year, month, date in this format. it doesn't help us any further to hard code the entire date, ie 2022-01-23T12:02:56.183Z.